experiment_status.md

```diff
@@ -1,3 +1,17 @@
+# Status 15-NOV-2024
+
+## Aaron's CoreCLR learnings
+
+- `PCODE` manifests itself from a myriad of places.
+  - The `MethodDesc::DoPrestub()` entry point should be referenced for many of the APIs that can produce `PCODE`. (for example, `GetStubForInteropMethod()`).
+- The ILStubResolver deletes the associated IL post JIT - `ILStubResolver::ClearCompileTimeState`.
+  - This means that when an IL stub will be the interpreted, clean-up of generated IL needs to be avoided.
```

Comment:
This is correct for the current CoreCLR interpreter that interprets IL directly. It is not correct for the current Mono interpreter that "JITs" the IL into its own bytecode.